spec/u3d/commands_spec.rb

```diff
@@ -265,11 +264,10 @@
         it 'resolves alias when passed as a version' do
           on_fake_os
           with_fake_cache('fakeos' => { 'versions' => { '1.2.3f4' => 'fakeurl' } })
+          definition = expected_definition('1.2.3f4', :fakeos, 'fakeurl')
```

Comment:
I'm not sure this is really a gain, as this doesn't reduce the number of lines and decreases readability in my opinion, as what goes into the fake cache is not clear.